uamqp/async_ops/session_async.py

```diff
@@ -46,7 +44,7 @@ def __init__(self, connection,
                  handle_max=None,
                  on_attach=None,
                  loop=None):
-        self.loop = loop or get_running_loop()
+        self._internal_kwargs = get_dict_with_loop_if_needed(loop)
```

Comment:
same comment as before on the default get_running_loop() behavior for python 3.6 -> 3.9